### DISCUSSION TITLE
Enable "Collection and search filters" via API

### DISCUSSION THREAD

**Discussion by patrick91** [Shopify/storefront-api-feedback#47] [Help]
Hello there! I've been trying to see if I could enable metafield filters via API. I think I found out how to create the metafields properly and now I'm only missing how to enable them via API

I've tried this query:

```graphql
mutation metafieldStorefrontVisibilityCreate($input: MetafieldStorefrontVisibilityInput!) {
  metafieldStorefrontVisibilityCreate(input: $input) {
    metafieldStorefrontVisibility {
      # MetafieldStorefrontVisibility fields
    }
    userErrors {
      field
      message
    }
  }
}
```

But I don't think it does what I need 😊

To be clear this is where I enable the filters manually:

<img width="983" alt="image" src="https://user-images.githubusercontent.com/667029/145557929-3b25897b-8df2-420b-affd-996b999463c9.png">

**Comment by evr-novicell** [comment]
Is this a duplicate of https://github.com/Shopify/storefront-api-feedback/discussions/10#discussioncomment-1782019 ?
